src/commandSocket.cpp

```diff
@@ -138,13 +138,14 @@ class CommandSocket::PathCompiler
         _layer_nr( 0 ),
         extruder(0),
         data_point_type(cura::proto::PathSegment::Point2D),
+        line_types(),
         line_widths(),
         points(),
         last_point{0,0}
     {}
     ~PathCompiler()
     {
-        if( points.size() )
+        if( line_types.size() )
```

Comment:
According to the code conventions there should be a space after the keyword `if` and it's not conventional to put extra spaces after an before the `(` and the `)`.